themes/classic/templates/catalog/_partials/miniatures/product.tpl

```diff
@@ -25,23 +25,23 @@
 {block name='product_miniature_item'}
   <article class="product-miniature js-product-miniature" data-id-product="{$product.id_product}" data-id-product-attribute="{$product.id_product_attribute}" itemscope itemtype="http://schema.org/Product">
     <div class="thumbnail-container">
-      {block name='product_thumbnail'}
-        {if $product.cover}
-          <a href="{$product.url}" class="thumbnail product-thumbnail">
-            <img
-              src = "{$product.cover.bySize.home_default.url}"
-              alt = "{if !empty($product.cover.legend)}{$product.cover.legend}{else}{$product.name|truncate:30:'...'}{/if}"
-              data-full-size-image-url = "{$product.cover.large.url}"
-            >
-          </a>
-        {else}
-          <a href="{$product.url}" class="thumbnail product-thumbnail">
-            <img
-              src = "{$urls.no_picture_image.bySize.home_default.url}"
-            >
-          </a>
-        {/if}
-      {/block}
+      <div class="product-thumbnail">
+        {block name='product_thumbnail'}
+          {if $product.cover}
+            <a href="{$product.url}" class="thumbnail">
```

Comment:
Changing the id breaks the tests (and potentially, other things). We should see if we can do this without changing the markup.

Reply:
I meant the class, sorry. I'm becoming more and more dyslexic as years go by 😁